pythonscript/bindings/builtins_binder/vector2.cpp

```diff
@@ -262,6 +262,19 @@ static mp_obj_t _make_new_vector2(const mp_obj_type_t *type, size_t n_args, size
 }
 
 
+static mp_obj_t _unary_op_vector2(mp_uint_t op, mp_obj_t in) {
+  auto self = static_cast<Vector2Binder::mp_godot_bind_t*>(MP_OBJ_TO_PTR(in));
+  switch (op) {
+    case MP_UNARY_OP_NEGATIVE:
```

Comment:
> Because I'm not creating new objects

But unary operator is not supposed to changed the object itself !
If you do `-Vector2()` it's not a trouble, but doing `v = Vector2(); -v` is...

Quick example in CPython:
```
(venv) >>> class V:
...     def __init__(self):
...         self.x = 1
...     def __neg__(self):
...         self.x = - self.x
...         return self
... 
(venv) >>> V()@
  File "<stdin>", line 1
    V()@
       ^
SyntaxError: invalid syntax
(venv) >>> V()
<__main__.V object at 0x7f2433d83e10>
(venv) >>> V().x
1
(venv) >>> (-V()).x
-1
(venv) >>> v = V()
(venv) >>> v.x
1
(venv) >>> -v
<__main__.V object at 0x7f2433d83e10>
(venv) >>> v.x
-1
(venv) >>> 
```